src/component/1d-2d/components/FloatMoleculeStructures/ActionsButton.tsx

```diff
@@ -5,7 +5,7 @@ import Button from '../../../elements/Button';
 
 export default function ActionsButton({ onFloatBtnClick }) {
   return (
-    <div data-no-export="true" style={{ display: 'flex', zIndex: 0 }}>
+    <div id="float-molecule-actions" style={{ display: 'flex', zIndex: 0 }}>
```

Comment:
It is not allowed to have multiple elements with the same id in html (this can happen when more than one molecule is floated)

Reply:
Yes, you are right, I will change it to class.

Reply:
You resolved it but it's still an id (and not unique).

Reply:
I'd keep an id, but include the uuid of the molecule in it to make it unique.

Reply:
Sorry, it is a mistake I did not commit the changes, I need to use class instead of id because i need still be able to access this element to get the height

Reply:
https://github.com/cheminfo/nmrium/pull/2091/commits/6eb32513f65b6ebabb7f52e27caa4f4a22ff905b